ImagePicker/RecordDurationLabel.swift

```diff
@@ -78,41 +58,36 @@ final class RecordDurationLabel : UILabel {
         indicatorLayer.opacity = 0
     }
     
-    // MARK: Private Methods
-    
-    private var secondTimer: Timer?
-    private var indicatorTimer: Timer?
+    private func initializeViews() {
+        indicatorLayer.masksToBounds = true
+        indicatorLayer.backgroundColor = UIColor(red: 234/255, green: 53/255, blue: 52/255, alpha: 1).cgColor
+        indicatorLayer.frame.size = CGSize(width: 6, height: 6)
```

Comment:
s it possible to abstract these values into configuration ?

Reply:
What configuration?

Reply:
put the configuration in enum, and use sttaic variables. the color could also be defined elsewhere in constatnts

Reply:
Dont agree, its one shot configuration, normally would be inside xib